### PR TITLE
Implement multiple choice question

### PR DESCRIPTION
**Pull request by iamquang95**

### CODE REVIEW

File: static/templates/learning/question.html [hunk 1]
```diff
@@ -32,13 +9,47 @@
     <div class="question-Detail well">
       <p class="question-Require"> Trả lời câu hỏi sau: </p>
       <p class="question-Text"> {{ vm.data.question }} </p>
+      <div ng-if = "vm.image != ''">
```
Comment (namnd95) [inline static/templates/learning/question.html]: This should compare with null as well.

Comment (iamquang95) [inline static/templates/learning/question.html]: Done!

File: static/templates/learning/question.html [hunk 2]
```diff
@@ -32,13 +9,47 @@
     <div class="question-Detail well">
       <p class="question-Require"> Trả lời câu hỏi sau: </p>
       <p class="question-Text"> {{ vm.data.question }} </p>
+      <div ng-if = "vm.image != '' || vm.image == null">
```
Comment (namnd95) [inline static/templates/learning/question.html]: && vm.image != null

Comment (iamquang95) [inline static/templates/learning/question.html]: ok

File: static/templates/learning/question.html [hunk 3]
```diff
@@ -32,13 +9,47 @@
     <div class="question-Detail well">
       <p class="question-Require"> Trả lời câu hỏi sau: </p>
       <p class="question-Text"> {{ vm.data.question }} </p>
+      <div ng-if = "vm.image != '' && vm.image != null">
```
Comment (namnd95) [inline static/templates/learning/question.html]: Should be vm.data.image

**Pull request merged**